src/ert/config/ert_config.py

```diff
@@ -822,9 +817,31 @@ def from_dict(cls, config_dict) -> Self:
         except ConfigValidationError as err:
             errors.append(err)
 
+        try:
+            analysis_config = AnalysisConfig.from_dict(config_dict)
```

Comment:
Does this mean that AnalysisConfig is created twice?

Reply:
I rearranged it because there was a chance that analysis_config would not exist when checking if it had a design_matrix